components/rmrk/utils.ts

```diff
@@ -228,14 +228,12 @@ export const getView = (rmrkString: string): RmrkMint | RmrkView | null => {
 }
 
 export const unwrap = (rmrkString: string): any | null => {
-  const rr = /{.*}/
-  const match = rmrkString.match(rr)
-
-  if (!match) {
+  try {
```

Comment:
It seems is not correct

```ts
a = 'RMRK::MINT::1.0.0::{a:1, b:2, c: "dr.dre" }'
const rr = /{.*}/
const x = a.match(rr)
console.log(x)
// will print: ["{a:1, b:2, c: \"dr.dre\" }"]
```

Reply:
okay. i thought the rmrkString is already an object. in this case i would have to reevaluate again an approach for this one